typify-impl/src/convert.rs

```diff
@@ -1505,4 +1538,143 @@ mod tests {
         let expected = quote! {};
         assert_eq!(actual.to_string(), expected.to_string());
     }
+
+    #[test]
+    fn test_nested_newtypes() {
```

Comment:
with regard to this test, I've added a new schema tester in `typify/tests/schemas` -- could you just add a new JSON Schema file there instead of this?

Reply:
Certainly, will do.